src/document/path/HolonomicPathStore.ts

```diff
@@ -197,15 +197,16 @@ export const HolonomicPathStore = types
         }
       );
     };
-    const setExporter = (exportFunction: (uuid: string) => void) => {
-      exporter = exportFunction;
+    /**Only to be used on the default path*/
```

Comment:
![image](https://github.com/user-attachments/assets/9de1d66c-227e-4454-839c-c62cbf0bf5a0)
The doc comment does show up when accessing the property, at least in VS Code.